app/facades/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  def self.ingredient_search(ingredient)
+    search_results = FoodService.search_by_ingredient(ingredient)
+    {
+      total_result_count: search_results[:totalHits],
+      top_ten_results: search_results[:foods].map { |result| Food.new(result) }
+    }
```

Comment:
Great job building a hash of the total results and your food objects to pass back to the controller here.